src/microbe_stage/PatchMapNameGenerator.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+
+public class PatchMapNameGenerator : IRegistryType
+{
+    [JsonRequired]
+    private int syllablesLowerLimit = 2;
+
+    [JsonRequired]
+    private int syllablesHigherLimit = 5;
+
+    private string continentName = null!;
+    private string patchName = null!;
+
+    [JsonRequired]
+    private List<string> syllables = null!;
+
+    [JsonRequired]
+    private List<string> sufixes = null!;
+
+    private string vowels = "aeiouy";
+    public string InternalName { get; set; } = null!;
+
+    public string GetContinentName()
+    {
+        return continentName;
+    }
+
+    public string GetPatchName()
+    {
+        return patchName;
+    }
+
+    /// <summary>
+    ///   Generates and returns a new patch name
+    /// </summary>
+    public string Next()
```

Comment:
This should allow passing in Random as a parameter so that one random can be used for the entire new game generation (so that we can move to allowing new save seeds at some point).